server/server/session/src/main/java/com/alipay/sofa/registry/server/session/push/PushEfficiencyImproveConfig.java

```diff
@@ -0,0 +1,267 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.registry.server.session.push;
+
+import com.alipay.sofa.registry.server.session.bootstrap.SessionServerConfig;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.HashSet;
+import java.util.Set;
+import org.apache.commons.collections.CollectionUtils;
+
+/**
+ * @author jiangcun.hlc@antfin.com
+ * @since 2023/3/16
+ */
+public class PushEfficiencyImproveConfig {
+
+  public static final int DEFAULT_CHANGE_TASK_WAITING_MILLIS = 100;
+  public static final int DEFAULT_PUSH_TASK_WAITING_MILLIS = 200;
+  public static final int DEFAULT_REG_WORK_WAITING_MILLIS = 200;
+
+  /** 三板斧 配置 当前IP */
+  private static String CURRENT_IP = "";
+  /** 三板斧配置 当前Zone */
+  private String CURRENT_ZONE;
+
+  static {
+    try {
+      CURRENT_IP = InetAddress.getLocalHost().getHostAddress();
+    } catch (UnknownHostException e) {
+    }
+  }
+
+  public int DEFAULT_CHANGE_DEBOUNCING_MILLIS = 1000;
+  public int DEFAULT_CHANGE_DEBOUNCING_MAX_MILLIS = 3000;
+  public int DEFAULT_PUSH_TASK_DEBOUNCING_MILLIS = 500;
+
+  /**
+   * session 收到 data 数据变更推送，创建 changeTask, delay changeDebouncingMillis 时间处理，避免数据连续变化触发大量推送, 默认
+   * 1000ms
+   */
+  private int changeDebouncingMillis = DEFAULT_CHANGE_DEBOUNCING_MILLIS;
+  /**
+   * session 收到 data 数据变更推送，创建 changeTask, delay changeDebouncingMillis 时间处理时，设置
+   * changeDebouncingMaxMillis，避免连续相同的任务一直刷新，使得 changeTask 饥饿得不到处理, 默认 3000ms
+   */
+  private int changeDebouncingMaxMillis = DEFAULT_CHANGE_DEBOUNCING_MAX_MILLIS;
+  /** session 异步处理 changeTask 的 Looper 间隔时间, 默认 100ms */
+  private int changeTaskWaitingMillis = DEFAULT_CHANGE_TASK_WAITING_MILLIS;
+  /** session 异步处理 pushTask 的 Looper 间隔时间, 默认 200ms */
+  private int pushTaskWaitingMillis = DEFAULT_PUSH_TASK_WAITING_MILLIS;
+  /** session 处理 pushTask delay pushTaskDebouncingMillis 时间处理，可以合并相同的推送任务，避免数据连续变化触发大量推送, 默认500ms */
+  private int pushTaskDebouncingMillis = DEFAULT_PUSH_TASK_DEBOUNCING_MILLIS;
+
+  /** Sub 请求的 BufferWorker Loop wait 时间, 默认 200ms */
+  private int regWorkWaitingMillis = DEFAULT_REG_WORK_WAITING_MILLIS;
+
+  /** 三板斧配置 配置在 Ip 集合 的 Session 机器生效, 要求大写 */
+  private Set<String> ipSet = new HashSet<>();
+
+  /** 三板斧配置 配置在 Zone 集合 的 Session 机器生效, 要求大写 */
+  private Set<String> zoneSet = new HashSet<>();
+
+  /** 三板斧配置 支持 按订阅方应用 设置 时间 */
+  private Set<String> subAppSet = new HashSet<>();
+
+  /** session 处理 pushTask delay pushTaskDebouncingMillis 时间处理，可以合并相同的推送任务，避免数据连续变化触发大量推送, 默认500ms */
+  private int sbfAppPushTaskDebouncingMillis = DEFAULT_PUSH_TASK_DEBOUNCING_MILLIS;
+
+  /**
+   * 判断是否满足 三板斧灰度条件
+   *
+   * @return
+   */
+  public boolean inIpZoneSBF() {
+    // 同时为空，全部生效
+    if (CollectionUtils.isEmpty(zoneSet) && CollectionUtils.isEmpty(ipSet)) {
```

Comment:
it will be effective to all cluster after registry deploy?

Reply:
after pushing configuration， if not set ipSet , zoneSet